client/lib/keyboard-shortcuts/menu.jsx

```diff
@@ -46,86 +46,83 @@ class KeyboardShortcutsMenu extends React.Component {
 		this.setState( { showDialog: ! this.state.showDialog } );
 	};
 
-	getShortcutsByCategory = () => {
-		var allShortcuts = KeyBindings.get(),
-			shortcutsByCategory = [
-				{
-					name: this.props.translate( 'List Navigation' ),
-					shortcuts: allShortcuts.listNavigation,
-					className: 'keyboard-shortcuts__list-navigation',
-					disabled: true,
-				},
-				{
-					name: this.props.translate( 'Site Navigation' ),
-					shortcuts: allShortcuts.siteNavigation,
-					className: 'keyboard-shortcuts__site-navigation',
-				},
-				{
-					name: this.props.translate( 'Reader' ),
-					shortcuts: allShortcuts.reader,
-					className: 'keyboard-shortcuts__reader',
-					disabled: true,
-				},
-				{
-					name: this.props.translate( 'Blog Posts and Pages' ),
-					shortcuts: allShortcuts.blogPostsAndPages,
-					className: 'keyboard-shortcuts__blog-posts-and-pages',
-					disabled: true,
-				},
-			];
+	getShortcutsByCategory() {
+		const allShortcuts = KeyBindings.get();
+		const shortcutsByCategory = [
+			{
+				name: this.props.translate( 'List Navigation' ),
+				shortcuts: allShortcuts.listNavigation,
+				className: 'keyboard-shortcuts__list-navigation',
+				disabled: true,
+			},
+			{
+				name: this.props.translate( 'Site Navigation' ),
+				shortcuts: allShortcuts.siteNavigation,
+				className: 'keyboard-shortcuts__site-navigation',
+			},
+			{
+				name: this.props.translate( 'Reader' ),
+				shortcuts: allShortcuts.reader,
+				className: 'keyboard-shortcuts__reader',
+				disabled: true,
+			},
+			{
+				name: this.props.translate( 'Blog Posts and Pages' ),
+				shortcuts: allShortcuts.blogPostsAndPages,
+				className: 'keyboard-shortcuts__blog-posts-and-pages',
+				disabled: true,
+			},
+		];
 
 		if ( config.isEnabled( 'devdocs' ) ) {
-			shortcutsByCategory = shortcutsByCategory.concat( {
+			shortcutsByCategory.push( {
 				name: this.props.translate( 'Developer' ),
 				shortcuts: allShortcuts.developer,
 				className: 'keyboard-shortcuts__developer',
 			} );
 		}
 
 		return shortcutsByCategory.map( function( category ) {
-			var classes = {};
-			classes[ 'keyboard-shortcuts__category' ] = true;
-			classes[ category.className ] = true;
-			classes[ 'keyboard-shortcuts__category-disabled' ] = category.disabled;
+			const className = classNames( 'keyboard-shortcuts__category', category.className, {
+				'keyboard-shortcuts__category-disabled': category.disabled,
+			} );
 
 			return (
-				<li className={ classNames( classes ) } key={ category.name }>
+				<li className={ className } key={ category.name }>
 					<h3>{ category.name }</h3>
 					<ul className="keyboard-shortcuts__list">
 						{ this.getShortcutList( category.shortcuts ) }
 					</ul>
 				</li>
 			);
 		}, this );
-	};
+	}
 
-	getShortcutList = shortcuts => {
-		return shortcuts.map( function( shortcut ) {
+	getShortcutList( shortcuts ) {
```

Comment:
Why are you changing the function from arrow one to the standard one?

Reply:
Methods should be the default and arrow function properties are useful only for methods that need to be auto-bound to `this`, because they are passed somewhere as event handlers.

The arrow function was most likely created by the React migration codemod, which doesn't know how the methods are used and always created arrow function properties, because that's safer.

Methods are slightly faster -- there is only one instance of the function on the prototype, while arrow properties need to be unique for each instance, and are created in constructor.